pandas/tests/indexes/period/test_astype.py

```diff
@@ -48,15 +49,17 @@ def test_astype_conversion(self):
         tm.assert_index_equal(result, expected)
 
         idx = period_range("1990", "2009", freq="A", name="idx")
-        result = idx.astype("i8")
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
```

Comment:
for a lot of these you can just change the test if you want to use view

Reply:
changed one of these, the rest were specifically targeting astype and not worth mucking with